guides/common/modules/snip_managing-packages-api.adoc

```diff
@@ -1,67 +1,69 @@
-Create a body of the API request in the JSON format by following the instructions below.
+Supply your API request in the JSON format.
 
 .API procedure
-. Create the `"job_invocation"` object and place rest of the body inside this object.
-. Create the `"inputs"` object with the `"package"` field of the string type specifying the packages you want to {PackageAction}.
-If you are specifying multiple packages, separate them with a whitespace.
-. Create a `"feature"` field of the string type with value `"katello_package_{PackageAction}"`.
-. Create a `"search_query"` field of the string type and input a search query matching the hosts on which you want to {PackageAction} the packages.
-. Optional: If you want to {PackageAction} packages as a specific user, create an `ssh` object with the following fields of the string type:
-* `"effective_user"` with the name of the ssh user
-* `"effective_user_password"` with the password of the ssh user if this password is required
-. Optional: If you want to {PackageAction} packages at a later time, create the `"scheduling"` object.
-The object can contain one or both of the following fields of the string type with date, time, and a timezone in the ISO 8601 format:
-* `"start_at"`- sets the time to {PackageAction} the packages
-* `"start_before"` - sets the latest time to {PackageAction} the packages.
-If it is not possible to {PackageAction} the packages by this time, then this action is cancelled.
 
+* Specify your API request:
```

Comment:
```suggestion
. Supply the API request body in the JSON format:
```
The API procedure contains two steps. Therefore, numbered list should be used. Also, this step defines the body. The actual API request is in the second step.